packages/shared/src/lib/core/activity/utils/helper/getGovernanceInfo.ts

```diff
@@ -32,7 +32,7 @@ export function getGovernanceInfo(output: Output, inputs: IWrappedOutput[], meta
                         ? GovernanceAction.IncreaseVotingPower
                         : GovernanceAction.DecreaseVotingPower,
                 votingPower: currentVotingPower,
-                votingPowerDifference: Math.abs(currentVotingPower - oldVotingPower),
```

Comment:
why are we removing absolute?

Reply:
We don't have `Math.abs()` for BigInt's.

Reply:
I created a `BigIntAbs()` function